### DISCUSSION TITLE
What is "horizontal" in "top, horizontal and bottom" ?

### DISCUSSION THREAD

**Discussion by BMJ-pdx** [kovidgoyal/kitty#3801] [Q&A]
(I seem to have gotten the answer to this -- "horizontal" is the divider between multiple kitty windows.  I was getting it confused with "horizontal sides".)
(Where's the Delete button? :-)

**Comment by kovidgoyal** [comment]
On Mon, Jul 05, 2021 at 07:05:51AM -0700, BMJ-pdx wrote:
> Should that be "vertical", not "horizontal"?  I.e., the left and right sides?
> If "horizontal" is indeed intended, how is it distinct from top and bottom?
> Thanks.

top margin, bottom margin and horizontal margins

**Reply by BMJ-pdx** [reply]
Horizontal margin is somewhat counterintuitive in this case, because "margin" usually refers to something at an edge.
And the distinction between margin, border, and padding is for the most part fuzzy until experimentation is done.
(Not a complaint or criticism, just an observation.  Tcl/Tk is much the same.)
